doc/json-rpc.md

```diff
@@ -227,3 +234,115 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1, "m
 ```
 
 ---
+
+### subscriptionChannel
+Open a socket on the node for JSON-RPC subscription requests
+
+##### Parameters:
+* `integer` - port to open for websocket subscription requests
```

Comment:
Why does the caller suggest a port?  It seems like it would be better if the node could choose the port to use and return it

Reply:
Sure, sounds like you prefer the first approach I mention above :)